db/migrate/20231025083153_create_users.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateUsers < ActiveRecord::Migration[7.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :post_counter
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Good job creating migration for the User , Post , Comment and Like , however to create only the migration is not enough, because you  `schema` fie is empty, Kindly make sure the schema file contains a schema that corresponds to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)